icu4c/source/i18n/number_decimfmtprops.cpp

```diff
@@ -15,6 +15,7 @@ using namespace icu::number::impl;
 
 namespace {
 
+alignas(DecimalFormatProperties)
```

Comment:
Adding Shane as a reviewer for this change.
(@sffc I made this change based on the comments in the Jira issue, as there was no existing commit to cherry-pick from.)

Reply:
Wouldn't it be cleaner to wait for the commit on master and then cherry-pick it from there?

Reply:
Yes, indeed it would be cleaner to wait for the commit/fix on master. That would also preserve authorship as well. (The commit on the maint branch would show up as 'authored' by Shane, and simply cherry-picked by myself).
I will revert this change -- actually I will just force push to the branch to remove the commit. 
Thanks Markus!

Reply:
The change is gone now.

Reply:
I made #63